src/utils/optionsdict.cc

```diff
@@ -128,7 +128,7 @@ class Lexer {
     }
 
     // Identifier
-    if (std::isalnum(str_[idx_])) {
+    if (std::isalnum(str_[idx_]) || str_[idx_]=='/') {
```

Comment:
Probably also `\` for those who use windows, and `.` for relative names.
Also I'm not sure whether `ReadIndentifier` accepts `.`, `/` `\`, and `.`, but if not, probably makes sense to do that too.

Reply:
`ReadIdentifier` accepts `_-./`, see <https://github.com/LeelaChessZero/lc0/blob/master/src/utils/optionsdict.cc#L171>. Unfortunately, `.` will not work as is here since it is already parsed with `ReadNumber`, it would need a more complicated lexer. However there is always `ReadString` for everything not caught here.